autotst/conformer/systematic.py

```diff
@@ -71,25 +71,46 @@ def find_all_combos(
     conformer.get_geometries()
 
     _, torsions = find_terminal_torsions(conformer)
-    cistranss = conformer.cistrans
-    chiral_centers = conformer.chiral_centers
 
     torsion_angles = np.arange(0, 360, delta)
     torsion_combos = list(itertools.product(
         torsion_angles, repeat=len(torsions)))
 
     if cistrans:
+        cistranss = []
```

Comment:
is the double `s` in `cistranss` a typo or just a plural of `cistrans`

Reply:
it's supposed to be the plural